openapi/components/schemas/AmlSettings.yaml

```
@@ -95,6 +95,46 @@ properties:
             $ref: ./AmlConfidence.yaml
           noCountry:
             $ref: ./AmlConfidence.yaml
+      cityAndRegionMatch:
+        type: object
+        description: Exact match of the customer's city and region of residence and exact name is found.
+        properties:
+          matchingCountry:
+            $ref: ./AmlConfidence.yaml
+          mismatchingCountry:
+            $ref: ./AmlConfidence.yaml
+          noCountry:
+            $ref: ./AmlConfidence.yaml
+      cityMatchAndRegionMismatch:
+        type: object
+        description: Exact match of the customer's city of residence and fuzzy, or inexact name match is found, but the region of residence does not match.
+        properties:
+          matchingCountry:
+            $ref: ./AmlConfidence.yaml
+          mismatchingCountry:
+            $ref: ./AmlConfidence.yaml
+          noCountry:
+            $ref: ./AmlConfidence.yaml
+      cityMatchAndNoRegion:
+        type: object
+        description: Exact match of the customer's city of residence and fuzzy, or inexact name match is found, but the region of residence is not provided.
```

suggestion: for readability
```suggestion
        description: Exact match of the customer's city of residence and fuzzy or inexact name match is found, but the region of residence is not provided.
```